cmd/lukso/flags.go

```diff
@@ -441,6 +468,7 @@ func prepareValidatorStartFlags(ctx *cli.Context) (startFlags []string) {
 	startFlags = clientDependencies[validatorDependencyName].PassStartFlags(ctx)
 
 	startFlags = append(startFlags, "--accept-terms-of-use")
+	startFlags = append(startFlags, fmt.Sprintf("--config-file=%s", ctx.String(validatorConfigFileFlag)))
```

Comment:
[maybe] It may be wrothwhile to resolve the path and provide absolute path

Reply:
Yeah, I could do that, but it doesn't seem to be that critical, so I will leave it in backlog while we resolve other higher-prio issues